spec/unit/helper_spec.rb

```diff
@@ -121,4 +121,18 @@ class Middleman::HashiCorpExtension
       expect(@instance.app.pretty_os("hashicorplinux")).to eq("Hashicorplinux")
     end
   end
+
+  describe "#latest_provider_version" do
+    before(:each) do
+      app = middleman_app
+      @instance = Middleman::HashiCorpExtension.new(app,
+        releases_enabled: false,
+      )
+      @instance.app = app
+    end
+
+    it "returns the latest provider version for a given provider" do
+      expect(@instance.app.latest_provider_version("template")).to eq("1.0.0")
+    end
+  end
```

Comment:
This is where the specs are for helper methods. Unfortunately I couldn't get them to run due to some middleman core issues that I don't think are worth the time to debug right now, but in theory this test will work fine if it ever does get patched. In the meantime, I fired up the nomad website locally, pointed to my local version via the Gemfile, and ran the method in a template, which worked as expected.